apps/payments/payments-next/app/api/hello/route.ts

```diff
@@ -0,0 +1,3 @@
+export async function GET(request: Request) {
+  return new Response('Hello, from API!');
+}
```

Comment:
Q: Instead of generic [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response), should we be using [`NextRequest`](https://nextjs.org/docs/pages/api-reference/functions/next-server#nextrequest) and [`NextResponse`](https://nextjs.org/docs/pages/api-reference/functions/next-server#nextresponse)?

```js
import type { NextRequest } from 'next/server';
import { NextResponse } from 'next/server';
```